iina/Base.lproj/MainMenu.xib

```diff
@@ -130,6 +130,9 @@
                                 </connections>
                             </menuItem>
                             <menuItem isSeparatorItem="YES" id="N0Z-h7-JSg"/>
+                            <menuItem title="Reveal Current File in Finder" id="che-dy-QUU">
```

Comment:
Similar to KeyBinding.strings, the MainMenu.strings in en.lproj must be updated with the new text.

Reply:
Am I supposed to do it manually or is there some sync command in XCode?

Reply:
See my latest comment on how I use `ibtool` to do it.